packages/faustwp-core/src/lib/getGraphqlEndpoint.ts

```diff
@@ -3,7 +3,7 @@ import { getWpUrl } from './getWpUrl.js';
 
 export function getGraphqlEndpoint(): string {
   const wpUrl = getWpUrl();
-  let graphqlEndpoint = `${wpUrl}/graphql`;
+  let graphqlEndpoint = `${wpUrl}/index.php?graphql`;
```

Comment:
This PR makes the existing GraphQL filter useless so we may want to follow up with a deprecation of https://faustjs.org/docs/plugin-system/filters#graphqlendpoint